llvm/include/llvm/IR/Instructions.h

```diff
@@ -3505,6 +3506,10 @@ class SwitchInst : public Instruction {
     return cast<BasicBlock>(getOperand(1));
   }
 
+  bool defaultDestIsUnreachable() const {
```

Comment:
Add documentation comment. I think the name is misleading too. The code isn't unreachable, and it only accepts empty blocks. How about something like defaultDestIsEmpty? defaultDestUndefined?

Reply:
Thanks for the reminder. I prefer `defaultDestUndefined`.

Reply:
https://github.com/llvm/llvm-project/commit/5932fcc47855fdd209784f38820422d2369b84b2#r138510755
cc @nikic